snapcraft/cli/store.py

```diff
@@ -32,8 +32,8 @@
     then we suggest you prefix the name with your developer identity,
     As ‘nessita-yoyodyne-www-site-content’.""")
 _MESSAGE_REGISTER_CONFIRM = dedent("""
-    We always want to ensure that users get the software they expect
-    for a particular name.
+    On behalf of the snapcraft store, we always want to ensure that
```

Comment:
can you please revert this? You will have to update many more things and it is leaving the scope of the bug.

While I appreciate the effort, this change will just make this PR take longer to land :wink:

Reply:
Okay sure

Reply:
Regarding, "You will have to update many more things", may I ask why?